packages/idx/src/property/search.component.html

```diff
@@ -14,7 +14,7 @@ <h4>Search Filters</h4>
 
     <div class="search-row search-basic clearfix">
         <div class="search-input search-location"
-             data-ng-class="{'location-input-active': !(options.query.where.City.length > 0 || options.query.where.CountyOrParish.length > 0 || options.query.where.MLSAreaMajor.length > 0 || options.query.where.Neighborhood.length > 0 || options.query.where.PostalCode.length > 0)}">
+             data-ng-class="{'location-input-active': !isPresetLocationSet()}">
```

Comment:
This is an excellent improvement in standards that avoids heavy resource parsing in favor of simple function calls.